src/en/clouds-manual.md

```diff
@@ -1,141 +1,212 @@
-automatically, but in the case of a manual cloud you will have to specify these
-resources.
+## Adding credentials
 
-This is done by using the `add-machine` command in Juju. This will install the
-default series on a given resource, and prepare it for
-deploying applications.
+Credentials should already have been set up via SSH. Nothing to do!
 
-For example, if you have a machine with the IP address 192.168.1.129, you could
-add it to your manual cloud with the command:
+## Creating a controller
+
+You are now ready to create a Juju controller for cloud 'mymanual':
+
+```bash
+juju bootstrap mymanual manual-controller
+```
+
+Above, the name given to the new controller is 'manual-controller'. The
+machine that will be allocated to run the controller on is the one specified
+during the `add-cloud` step. In our example it is the machine with address
+10.143.211.93.
+
+For a detailed explanation and examples of the `bootstrap` command see the
+[Creating a controller][controllers-creating] page.
+
+## Adding machines to a Manual cloud
+
+To add the machine with an IP address (and user) of bob@10.55.60.93 to the
+'default' model in the Manual cloud (whose controller was named
+'manual-controller'):
 
 ```bash
-juju add-machine ssh:ubuntu@192.168.1.129
+juju add-machine ssh:bob@10.55.60.93
 ```
 
-In this case, assuming SSH is available and there is a user called 'ubuntu' on
-that machine, you will be prompted for a password. As the preparation requires
-sudo privileges on the remote machine, you will be asked for the password once
-again as the installer initialises.
+Unless you're using passphraseless public key authentication, you may be
+prompted for a password a few times. The process takes a couple of minutes.
 
-The process should not take too long, and you will see messages on the screen as
-the various stages progress. Once the command has returned, you can check
-the machine is available by running:
+Once the command has returned, you can check that the machine is available:
 
 ```bash
-juju status
+juju machines
 ```
 
-## Deploying
+Sample output:
+
+```no-highlight
+Machine  State    DNS          Inst id             Series  AZ  Message
+0        started  10.55.60.93  manual:10.55.60.93  xenial      Manually provisioned machine
+```
 
-Usually Juju will create machines as it needs them. By default, using a charm
-to deploy an application will automatically create a machine first and then
-deploy the new application.
+## Deploying a charm in a Manual cloud
 
-With a manual cloud, you need to create your machines in advance, so Juju will
-need to know which of the machines you wish to target when you deploy a charm
-(or scale out). This can be done with the use of the
-[placement directives][placement]: you need to use the
-'--to' option to specify the destination, which is either a machine or a
-container on a machine:
+To deploy WordPress onto the machine we added previously its ID (of '0') is
```

Comment:
I'd maybe rewrite the 'its ID' part, or add a comma. eg. *, we need to declare the ID (of '0') of the machine.*

Reply:
:heavy_check_mark: